src/NovelRT/Ecs/Audio/AudioSystem.cpp

```diff
@@ -17,6 +17,18 @@ namespace NovelRT::Ecs::Audio
         _service->InitializeAudio();
     }
 
+    AudioSystem::AudioSystem(bool verbose)
```

Comment:
Can this constructor throw?

Reply:
Maybe?

Reply:
This ctor has been removed ,but the original _might_ throw since we make a call to the service (which throws)

Reply:
Gotcha.